src/public/constants.ts

```diff
@@ -4,6 +4,7 @@ export enum HostClientType {
   android = 'android',
   ios = 'ios',
   rigel = 'rigel',
+  surfaceHub2 = 'surfaceHub2',
```

Comment:
As discussed in our sync please change this to the more generic `surfaceHub` instead of `surfaceHub2`.

Reply:
Changed.